src/neuroglancer/datasource/graphene/frontend.ts

```diff
@@ -577,6 +873,30 @@ class GrapheneGraphSource extends SegmentationGraphSource {
     return this.graphServer.getRoot(segment);
   }
 
+  tabContents(layer: SegmentationUserLayer) {
+    return new DependentViewWidget(
```

Comment:
I'm using 2 DependentViewWidgets. One in the SegmentationGraphSourceTab, the other provided by the graph implementation. It seemed to make sense here rather than trying to just return an element because I need the graph and the context.